tests/containers/search/topFilterBar/filterGroups/AgencyFilterGroup-test.jsx

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
We don't normally write unit tests for UI components, but since these have some logic for creating the labels I think it's a good idea. The two test files can be moved to `tests/components/search/filters/agency`

Reply:
Well, that and I'm trying to learn.  :-p